### PR TITLE
OSD select sort

### PR DESCRIPTION
**Pull request by HThuren**
Now sorted the selects at element list and timer source and precision

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Comment by HThuren** [comment]
Thank you

**Comment by SJurick** [comment]
Thank you

**Pull request merged**
